Crocopio emulates keyboard.

nanusefue opened the pull request.

cuvoodoo commented:
here some remarks:
- the next available PID would be 0x613d instead of 0xcc16 (0xcc15 was a special request)
- the USB ID entry should include a link to the project
- the https://github.com/nanusefue/crocopoi/ repository includes a LICENSE file for the GPLv3 licence, but no source code is available (only a binary firmware is available)
- the hardware part must also include the exports (PDF schematic, gerbers, NC drill, BOM, ...). this allows others not having the CAD for the source files to have a look at the project, and me to verify if is it really open source

nanusefue commented:
I have already added all the requirements that have requested me.\
Thank you very much for the response.

cuvoodoo commented:
thanks for the very fast fixes. here a couple more remarks (mostly about licenses):
- could you place the USB ID line change after the line with PID 0x613c in the file to keep the order
- the hardware files do not include license information. since the GPLv3 license has been designed for source code and software, I don't assume you are using this license for the hardware design files. alternative licenses are CERN OHL, Creative Commons, if you want to stay with GNU licenses the GNU Free Documentation License, etc. . nevertheless, could you please add the license name in the hardware files (most of the time in the schematic, in the bottom right sheet information corner), and the corresponding license file in the folder
- your project seem to use source code for other projects (mainly V-USB, but I did not check everything). Please add a summary of the projects you use, and corresponding license information (often in the README, LICENSE, or CONTRIBUTION file)
- to clearly identify which code is under which license (particularly when other projects are included), it is recommended to add a license header in the corresponding source file (i.e. Crocopoi.ino, main.c). See https://www.gnu.org/licenses/gpl-howto.html for good practice.

nanusefue commented:
Thanks for the help it is the first time that I correctly document an OpenSource project, I hope that now everything is correct.
- He  has added the hardware license accordingly, and has also added it in the schematics.
- I added the v-usb license in the contribute, in addition to having its own license in the v-usb project folder
- I  have modified the source code by adding the header indicating the type of license.

Thank you so much

cuvoodoo commented:
It now is at least good enough for me to grant you a USB PID. Congrats for your project.
I see you chose the CERN OHL. If you want to correctly apply this license, I recommend you to have a look at their [howto](https://www.ohwr.org/attachments/2392/cern_ohl_v_1_2_howto.pdf). It requires a few more files which also help to nicely document the hardware project.
Last note: your commit adds a new empty line. This might be because you delete the last line, or your editor adds empty last line.
